vars/perfRegressionParallelPipeline.groovy

```diff
@@ -218,14 +218,27 @@ def call(Map pipelineParams) {
                                                 }
                                             }
                                         }
-                                        stage('Clean SCT Runners') {
-                                            steps {
-                                                catchError(stageResult: 'FAILURE') {
-                                                    script {
-                                                        wrap([$class: 'BuildUser']) {
-                                                            dir('scylla-cluster-tests') {
-                                                                cleanSctRunners(params, currentBuild)
-                                                            }
+                                        stage("Send email for ${sub_test}") {
+                                            catchError(stageResult: 'FAILURE') {
+                                                script {
+                                                    wrap([$class: 'BuildUser']) {
+                                                        dir('scylla-cluster-tests') {
+                                                            def email_recipients = groovy.json.JsonOutput.toJson(params.email_recipients)
+
+                                                            sh """
+                                                            #!/bin/bash
+
+                                                            set -xe
+                                                            env
+
+                                                            test_id=`cat latest/test_id`
+                                                            echo "start check_regression and send email ..."
+                                                            es_id=`cat \${test_id}_es_doc_id`
```

Comment:
if test will be run on sct-runner, then it is fine to cat from latest/test_id,
if it will be run on builder, we can get sitiuation that get result for wrong test.